src/main/webapp/nav.html

```diff
@@ -4,6 +4,6 @@
         <li><a href="index.html">Seniors Connect</a></li>
         <li><a href="index.html">Home</a></li>
         <li><a href="techassistant.html">Tech Assistant</a></li>
-        <li><a id="login" href="login.html">Login</a></li>
+        <li><a id="login" href="login.html">INCORRECT</a></li>
```

Comment:
Is this the link for logging in or logging out? Seems a little confusing to have INCORRECT and the id of login.

Reply:
INCORRECT was added to make sure updating Login/Logout works properly. If we set it to "login" at the start, and for some reason the javascript fails, we will have a functionless "login" button.

Reply:
Ok that makes sense.  Thanks :)